### PR TITLE
Updated readme file for login

### PR DESCRIPTION
**Pull request by rdhawladar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,45 @@
...
+    |-- go.sum              # Contains the expected cryptographic checksums of the content of specific module versions
+    |-- Makefile            # Makefile to run commands after docker up
+    |-- readme.md           # Explains project installation and other informations
+
+## Tools and Technology
+    Golang
+    PostgreSQL
+
+## Installation
+### Step-1
```
Comment (shuza) [inline README.md:43]: better to add single command like `make copy-config` using makefile

File: README.md [hunk 2]
```diff
@@ -11,10 +11,45 @@
...
+    Copy/rename `.env.example` file as `.env`. Change the `APP_PORT`, `DB_PORT`, `DB_NAME`,`DB_HOST, DB_USER, DB_PASSWORD` value as per your DB and Project setup.
+
+### Step-2
```
Comment (shuza) [inline README.md:46]: better to add single command like `make migrate` using makefile

File: README.md [hunk 3]
```diff
@@ -11,10 +11,45 @@
...
+    Migrate DB using the following command:
+    ```bash
+    go run main.go migrate
+    ```
+### Step-3
```
Comment (shuza) [inline README.md:51]: better to add single command like `make server` using makefile

**Review by shuza — CHANGES_REQUESTED** [review]
use makefile to make steps simpler

**Comment by zrshishir** [comment]
It looks good to me

**Review by zrshishir — APPROVED** [review]
It looks good to me

**Review by zrshishir — COMMENTED** [review]
Project structure, Tools and Technology and Installation section should be added before the Feature Details section

**Review by shuza — APPROVED** [review]
lgtm. it will be better. if we can simplify commands using `Makefile` but we can do it in separate PR. I have create and issue for that https://github.com/teachmind/Auth-service/issues/14

**Pull request merged**